frontend/src/reviews/ReviewRoleDisplay.tsx

```diff
@@ -25,7 +25,7 @@ export default function ReviewRoleDisplay({ review }: ReviewRoleDisplayProps) {
   return (
     <>
       {isModelRolesLoading && <Loading />}
-      <Stack direction='row' alignItems='center' justifyContent='center' spacing={1}>
+      <Stack direction='row' alignItems='center' justifyContent='center' spacing={1} data-test='reviewRoleMessage'>
```

Comment:
Looks like this `data-test` isn't being used in your tests so it can be removed.